lib/Middleware/State.php

```diff
@@ -238,8 +238,8 @@ public static function setState(App $app, Request $request): Request
 
         // Configure logging
         if (Environment::isForceDebugging() || strtolower($mode) == 'test') {
-            error_reporting(E_ALL);
-            ini_set('display_errors', 1);
+           // error_reporting(E_ALL);
```

Comment:
Did you mean for this to be commented out in the PR?

Reply:
oops, no I didn't, will bring this back in the next PR, thanks.